SampleProject/src/autonomous.cpp

```diff
@@ -23,6 +23,8 @@ void Autonomous::execute(int selection) {
 void Autonomous::runAutonomus1() {
  robot.driveFor(forward, 11, distanceUnits::in);
  robot.turnFor(right, 90, degrees);
+ // I am another peter
+ robot.driveFor(forward, 2, distanceUnits::in);
```

Comment:
Ok.. I think we need to convert constant value 2 to a variable like distanceToGoal.